src/transformers/models/altclip/processing_altclip.py

```diff
@@ -16,10 +16,33 @@
 Image/Text processor class for AltCLIP
 """
 
-import warnings
+from typing import List, Union
 
-from ...processing_utils import ProcessorMixin
-from ...tokenization_utils_base import BatchEncoding
+
+try:
+    from typing import Unpack
+except ImportError:
+    from typing_extensions import Unpack
```

Comment:
Rather than try/except could we do this in a more explicity way e.g. by testing on the version in the env?

Reply:
done!